data/139/359/646/5/1393596465.geojson

```diff
@@ -18,91 +18,151 @@
     "lbl:longitude":9.78019,
     "lbl:max_zoom":18.0,
     "mz:hierarchy_label":1,
-    "mz:is_current":-1,
+    "mz:is_current":1,
     "mz:min_zoom":15.0,
     "mz:note":"Quattroshapes Point Gazetteer import 20170519",
     "name:als_x_preferred":[
+        "Gamerschwang"
+    ],
+    "name:als_x_variant":[
         "Balderschwang"
     ],
     "name:ceb_x_preferred":[
+        "Gamerschwang"
+    ],
+    "name:ceb_x_variant":[
         "Balderschwang"
     ],
     "name:deu_x_preferred":[
+        "Gamerschwang"
+    ],
+    "name:deu_x_variant":[
         "Balderschwang"
     ],
     "name:epo_x_preferred":[
+        "Gamerschwang"
+    ],
+    "name:epo_x_variant":[
         "Balderschwang"
     ],
     "name:eus_x_preferred":[
+        "Gamerschwang"
+    ],
+    "name:eus_x_variant":[
         "Balderschwang"
     ],
-    "name:fas_x_preferred":[
+    "name:fas_x_variant":[
```

Comment:
Why are we setting variants but not preferred?

Reply:
This PR moves all existing preferred names variant names. Then, when available, new preferred names were added.

With this language (and the other one below), I'm unable to find a `"name:fas_x_preferred"` name because it isn't a Latin-based language and not something I can find online (like on Wikidata, for example).

So in a handful of cases, this record will only have a variant name for a language.

Reply:
Huh, ok